src/app/components/configure-app/configure-app.component.ts

```diff
@@ -137,6 +144,29 @@ export class ConfigureAppComponent implements OnInit {
 
   async ngOnInit() {
     this.loadFromSettings();
+    this.updateNodeStats();
+  }
+
+  async updateNodeStats(refresh=false) {
+    if ((this.serverAPIUpdated != this.appSettings.settings.serverAPI && this.selectedServer === 'random') || (refresh && !this.statsRefreshEnabled)) return
+    this.statsRefreshEnabled = false;
+    try {
+      let blockCount = await this.api.blockCount()
+      this.nodeBlockCount = this.util.string.addCommas(blockCount.count.toString())
+      this.nodeUnchecked = this.util.string.addCommas(blockCount.unchecked.toString())
+      this.nodeCemented = this.util.string.addCommas(blockCount.cemented.toString())
```

Comment:
Why not use toLocaleString? 
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/toLocaleString

This helps us later when we get to translations as well, then we would need to format the numbers according to the locale you chose.

Reply:
Mostly because the rest of the site is in English. Many countries, Sweden for example, format a large number by using dot instead of comma. Such as 123.456 and that is really confusing when the rest of the site is using dot as a decimal separator. It does make sense when we get translation working so I will change it to use toLocaleString('en-US') for now.